store/sqlstore/user_store.go

```diff
@@ -592,9 +596,30 @@ func (us SqlUserStore) GetAllProfilesInChannel(channelId string, allowFromCache
 	if err != nil {
 		return nil, model.NewAppError("SqlUserStore.GetAllProfilesInChannel", "store.sql_user.app_error", nil, err.Error(), http.StatusInternalServerError)
 	}
-
 	var users []*model.User
-	if _, err := us.GetReplica().Select(&users, queryString, args...); err != nil {
+	rows, err := us.GetReplica().Db.Query(queryString, args...)
+	if err != nil {
+		return nil, model.NewAppError("SqlUserStore.GetAllProfilesInChannel", "store.sql_user.get_profiles.app_error", nil, err.Error(), http.StatusInternalServerError)
+	}
+
+	defer rows.Close()
+	for rows.Next() {
+		var user model.User
+		var propsBytes []uint8
+		var notifyPropsBytes []uint8
+		var timezoneBytes []uint8
+		err = rows.Scan(&user.Id, &user.CreateAt, &user.UpdateAt, &user.DeleteAt, &user.Username, &user.Password, &user.AuthData, &user.AuthService, &user.Email, &user.EmailVerified, &user.Nickname, &user.FirstName, &user.LastName, &user.Position, &user.Roles, &user.AllowMarketing, &propsBytes, &notifyPropsBytes, &user.LastPasswordUpdate, &user.LastPictureUpdate, &user.FailedAttempts, &user.Locale, &timezoneBytes, &user.MfaActive, &user.MfaSecret, &user.IsBot, &user.BotDescription, &user.BotLastIconUpdate)
+		if err != nil {
+			mlog.Debug("Cannot scan sql row into struct fields", mlog.Err(err))
```

Comment:
I would just return an AppError from here rather than trying to log. That's what gorp does too - returns directly if it encounters an error during scanning.

So it may be better to remain consistent, rather than ignore errors from some rows and return partial results.